README.md

```diff
@@ -1,13 +1,12 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+##Obi Abii's __portfolio__
 
-To get started:
+This repo contains Obi's *STEP projects*. As of now, I have implemented 0 of the projects;
+###The Projects
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
+1. NaN		0%
+2. NaN		0%
+3. NaN		0%
+4. NaN		0%
```

Comment:
I used NaN as a placeholder because I didn't know what the projects would be called.

Reply:
Great! Often we use TODO comments to let us and others know we will or should address something in the future.

Reply:
Sure thing, I'll add those!